Dockerfile.metering-ansible-operator

```diff
@@ -1,34 +1,30 @@
 # need the helm-cli from the helm image
 FROM quay.io/openshift/origin-metering-helm:latest as helm
```

Comment:
While we're at it, we may as well change this to use the corresponding registry.svc.ci.openshift.org OCP imagestream.